js/src/contracts/registry.js

```diff
@@ -101,4 +101,17 @@ export default class Registry {
       return address;
     });
   }
+
+  lookupMeta (_name, key) {
```

Comment:
Ah yes good call. Well I guess that's fine, it small enough. Could have a

```
getLookupParams (_name, key) {
  const name = _name.toLowerCase();
  const sha3 = this._api.util.sha3.text(name);

  return [sha3, key];
}
```

method though